### PR TITLE
P0771R1 std::function move constructor should be noexcept

### LINKED ISSUE #2422: [2018-11 LWG Motion 16] P0771R1 std::function move constructor should be noexcept
[P0771R1](http://wg21.link/p0771r1)

### PR DESCRIPTION
**Pull request by burblebee**
Fixes #2422.

### CODE REVIEW

File: source/utilities.tex [hunk 1]
```diff
@@ -14554,7 +14554,7 @@
     function() noexcept;
     function(nullptr_t) noexcept;
     function(const function&);
-    function(function&&);
+    function(function&&) noexcept;;
```
Comment (Eelis) [inline source/utilities.tex]: Extra semicolon

File: source/utilities.tex [hunk 2]
```diff
@@ -14676,7 +14676,7 @@
 
 \indexlibrary{\idxcode{function}!constructor}%
 \begin{itemdecl}
-function(function&& f);
+function(function&& f) noexcept;;
```
Comment (Eelis) [inline source/utilities.tex]: Extra semicolon here, too.

File: source/utilities.tex (hunk 1 above)
Comment (burblebee) [reply source/utilities.tex]: thanks for catching this cut&paste error!

File: source/utilities.tex (hunk 2 above)
Comment (burblebee) [reply source/utilities.tex]: ditto!

**Pull request merged**